backend/src/services/friendNotification.ts

```diff
@@ -1,25 +1,44 @@
-export const friendRequestNotification = {
-  "notification": {
-    "title": "Boop Friend Request!",
-    "body": "You got a friend request on Boop",
-    // TODO add an icon that works even when the app is closed
-    "silent": false,
-    "actions": [{
-      "action": "show_friends_page",
-      "title": "View friend request"
-    }]
-  }
-};
 
-export const friendAcceptNotification = {
-  "notification": {
-    "title": "Boop Friend Request!",
-    "body": "Someone accepted your friend request, check out who that was",
-    // TODO add an icon that works even when the app is closed
-    "silent": false,
-    "actions": [{
-      "action": "show_friends_page",
-      "title": "View Friends"
-    }]
+import { database } from "../services/database";
+import { throwBoopError } from "../util/handleAsync";
+
+
+async function queryFullName(fromUUID: string): Promise<string> {
+  const rows = await database.query<{full_name: string;}>(
+    `select full_name from users where user_uuid = $1`, [fromUUID]
+  );
+  if (rows.length === 0) {
+    throwBoopError("User not found", 404);
   }
-};
+  return rows[0].full_name;
+}
+
+export async function friendRequestNotification(fromUUID: string): Promise<{notification: Object;}> {
+  const senderUsername: string = await queryFullName(fromUUID);
```

Comment:
i recommend changing this variable name to `senderName` or `senderFullName` since you're no longer using usernames,